src/python/pants/backend/docker/goals/package_image.py

```diff
@@ -334,6 +336,18 @@ def get_build_options(
             yield from target[field_type].options()
         elif issubclass(field_type, DockerBuildOptionFlagFieldMixin):
             yield from target[field_type].options()
+        elif issubclass(field_type, DockerImageBuildImageCacheToField) or issubclass(
+            field_type, DockerImageBuildImageCacheFromField
+        ):
+            source = InterpolationContext.TextSource(
```

Comment:
This copies the same interpolation context / formatter used elsewhere - I think for some of the [cache backends](https://docs.docker.com/build/cache/backends/s3/) it would be useful to interpolate env vars. 

But I'm not sure it's worth refactoring (or how) at this stage, as potentially some of the values used for repository interpolation could be useful as well - so it's possible this could get extended.